next.config.js

```diff
@@ -84,7 +84,7 @@ const moduleExports = {
   reactStrictMode: true,
   images: {
     loader: "default",
-    domains: ["localhost", "cmsmediaproduction.s3.amazonaws.com", "cmsmediastaging.s3.amazonaws.com", "avatars.githubusercontent.com"],
+    domains: ["localhost", "0.0.0.0", "cmsmediaproduction.s3.amazonaws.com", "cmsmediastaging.s3.amazonaws.com", "avatars.githubusercontent.com"],
```

Comment:
I'm just wondering if we should make this an environment variable or even a simple constant object that has appropriate domains for each environment (local, staging and prod)

Reply:
I agree that would be cleaner. If we need to modify this line one more time let's go ahead and clean it up, leaving as is for now.